pkg/local_object_storage/metabase/control.go

```diff
@@ -1,14 +1,19 @@
 package meta
 
 import (
+	"errors"
 	"fmt"
 	"path/filepath"
 
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/shard/mode"
 	"github.com/nspcc-dev/neofs-node/pkg/util"
 	"go.etcd.io/bbolt"
 	"go.uber.org/zap"
 )
 
+// ErrDegradedMode is returned when metabase is in a degraded mode.
+var ErrDegradedMode = errors.New("metabase is in a degraded mode")
```

Comment:
isnt "meta" and "degraded" a confusing combination of words?

Reply:
to me they are not

Reply:
ok, mb that is my problem: i thought that "degraded" was "without meta", and that is why i look at that it "meta without meta"